irohad/main/iroha_conf_loader.hpp

```diff
@@ -28,6 +28,24 @@ namespace config_members {
   const char *MstSupport = "mst_enable";
 }  // namespace config_members
 
+static constexpr size_t kBadJsonPrintLength = 15;
+static constexpr size_t kBadJsonPrintOffsset = 5;
+static_assert(kBadJsonPrintOffsset <= kBadJsonPrintLength,
+              "The place of error is out of the printed string boundaries!");
+
+std::string reportJsonParsingError(const rapidjson::Document &doc,
+                                   const std::string &conf_path,
+                                   std::istream &input) {
+  const size_t error_offset = doc.GetErrorOffset();
+  const size_t print_offset =
+      std::max(error_offset, kBadJsonPrintOffsset) - kBadJsonPrintOffsset;
```

Comment:
Quite non obvious expression, where is that comes from?

Reply:
It computes the number of symbols we can take before the error position without crossing the buffer boundary.

Reply:
Maybe comment it then? It is really not very understandable